### PR TITLE
Sass2

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

**Comment by pmuir** [comment]
Please wait and see if tests will pass.

**Comment by michaelkleinhenz** [comment]
Looks like there is something fishy with the loaders..

**Comment by pmuir** [comment]
Yep, fixing.

File: config/webpack.common.js [hunk 1]
```diff
@@ -82,36 +88,59 @@ module.exports = {
...
+        loader: extractCSS.extract({
+          fallback: "style-loader",
+          use: "css-loader?sourceMap&context=/"
+        })
       },
+
       {
         test: /\.scss$/,
-        use: [
+        loaders: [
```
Comment (joshuawilson) [inline config/webpack.common.js]: Why change this to "loaders", under webpack 2.2 it should be "use". The allow "loaders" for backwards compatibility but it is no longer the standard.

**Comment by joshuawilson** [comment]
@michaelkleinhenz if you approve it, I can merge it without adding a Merge commit. I confirmed there are not merge conflicts.

**Pull request merged**